truss/cli/cli.py

```diff
@@ -756,8 +756,10 @@ def run_python(script, target_directory):
                 options["fg"] = "red"
 
             click.secho(output_content.decode("utf-8", "replace"), nl=False, **options)
+        exit_code = container.wait()
+        sys.exit(exit_code)
     except DockerException:
-        pass
+        sys.exit(1)
```

Comment:
can we make a ticket to explore extracting the right exit code from DockerException and propogating it?

Reply:
i actually think we can remove this try/except now